packages/x-components/src/x-modules/facets-next/events.types.ts

```diff
@@ -0,0 +1,45 @@
+import { Facet } from '@empathyco/x-types';
```

Comment:
import the `Facet` from `@empathyco/x-types-next`

Reply:
Buen ojo